include/deal.II/grid/reference_cell.h

```diff
@@ -58,6 +58,41 @@ namespace internal
      */
     DEAL_II_CONSTEXPR dealii::ReferenceCell
                       make_reference_cell_from_int(const std::uint8_t kind);
+
+
+    // Define VTK constants for linear, quadratic and
+    // high-order Lagrange geometrices
+    enum vtk_cell_types
```

Comment:
I would prefer if you moved the declaration of this enum into the `.cc` file and if these were members of an `enum class VTKCellTypes`.

Reply:
More precisely, move it into an anonymous namespace.

Reply:
I did that, but now there is not implicit conversion to unsigned it, so I need to use static_cast<unsigned int>(...). Is that ok?

Reply:
Ah, I see. That's ugly as well...

How about
```
  namespace {
    namespace VTKCellTypes {
      constexpr unsigned int  VTK_abc = 42;
    }
  }
```
?

Reply:
Or
```
  namespace {
    namespace VTKCellTypes {  
      enum {    // no name given
        VTK_abc = 42
     };
  }
}
```

Reply:
I'm fine doing it either way, but I'm curious why not 

```
 namespace {
      enum VTKCellTypes {   
        VTK_abc = 42
     };
  }
```

Reply:
enums are not scoped and neither are anonymous namespaces. Using your example, you inject the name into the global namespace of that file. We try to avoid that and apply some basic "sanitary practices" to the global namespace :-)